### PR TITLE
DDF-2804 Update hardening guide to recommend 10 minute session timeout

### PR DESCRIPTION
**Pull request by mcalcote**
#### What does this PR do?
Added a row to the Environment Hardening suggestions that describes how to update the Session Timeout to the recommended 10 minutes.

#### Who is reviewing it (please choose AT LEAST two reviewers that need to approve the PR before it can get merged)?
@ricklarsen @ahoffer 

#### Select at least one member from relevant component team(s) from below (at least one component team member needs to approve the PR).
[Docs](https://github.com/orgs/codice/teams/docs) @Lambeaux 

#### Choose 2 committers to review/merge the PR (please choose ONLY two committers from below, delete the rest).
@shaundmorris
@stustison

#### How should this be tested? (List steps with links to updated documentation)
Read the changes, observe the generated table.

#### What are the relevant tickets?
[DDF-2804](https://codice.atlassian.net/browse/DDF-2804)

#### Screenshots (if appropriate)
screen shot 2017-02-14 at 4 07 10 pm

screen shot 2017-02-14 at 5 22 13 pm
(PDF)

#### Checklist:
- [x] Documentation Updated
- [ ] Change Log Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

### CODE REVIEW

File: distribution/docs/src/main/resources/_contents/_securing/environment-hardening-contents.adoc [hunk 1]
```diff
@@ -62,4 +62,10 @@ a|Update the `<${branding}_HOME>/etc/org.ops4j.pax.web.cfg` file to add the entr
 Setting this configuration may break compatibility to legacy systems that do not support two-way SSL.
 ====
 
+|Session Inactivity Timeout
+|unauthorized access
+a|Update the Session configuration to have no greater than 10 minute Session Timeout.
+
+ * To set the timeout, navigate to ${ddf-security}
```
Comment (ricklarsen) [inline distribution/docs/src/main/resources/_contents/_securing/environment-hardening-contents.adoc]: Can you revise the steps to
`* Navigate to the *${admin-console}.`
`* Select the *${ddf-security}* application.`
`* Select the *Configuration* tab.
`* Select *Session*.`
`* Set *Session Timeout (in minutes)* to `10` (or less).`

Comment (mcalcote) [inline distribution/docs/src/main/resources/_contents/_securing/environment-hardening-contents.adoc]: Thank you, Revised

File: distribution/docs/src/main/resources/_contents/_securing/environment-hardening-contents.adoc [hunk 2]
```diff
@@ -62,4 +62,13 @@ a|Update the `<${branding}_HOME>/etc/org.ops4j.pax.web.cfg` file to add the entr
 Setting this configuration may break compatibility to legacy systems that do not support two-way SSL.
 ====
 
+|Session Inactivity Timeout
+|unauthorized access
+a|Update the Session configuration to have no greater than 10 minute Session Timeout. +
```
Comment (emmberk) [inline distribution/docs/src/main/resources/_contents/_securing/environment-hardening-contents.adoc]: nit: than **a** 10 minute timeout

Comment (mcalcote) [inline distribution/docs/src/main/resources/_contents/_securing/environment-hardening-contents.adoc]: Updated

**Comment by mcalcote** [comment]
Added periods to the other directions on this table 👍

**Pull request merged**